ProjectEureka/project-eureka-flutter
# PE-34 | Implement Main Settings Page

**WHAT I DID:**

- Settings page with links to General, Payment, and Account settings screens
- Log Out button that log users out and navigates them the login page

**HOW TO TEST:**

1. Navigate to Settings page through Side Menu
2. Click Logout. Terminal must confirm that user is logged out. It must navigate to Login page. "Back" button must not function as the user is not logged in anymore.
3. Navigation through Settings screens must work as intended
